SecurityPkg/Tcg/Tcg2Pei/Tcg2Pei.c

```diff
@@ -42,14 +42,15 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include <Library/ResetSystemLib.h>
 #include <Library/PrintLib.h>
 
-// MU_CHANGE_23086
-// MU_CHANGE [BEGIN] - Add the OemTpm2InitLib
+// MU_CHANGE [BEGIN] - 23086 - Add the OemTpm2InitLib
 #include <Library/OemTpm2InitLib.h>
 // MU_CHANGE [END]
-// MU_CHANGE_131467
-// MU_CHANGE [BEGIN] - Move to 256-bit PCRs.
+// MU_CHANGE [BEGIN] - 131467 - Move to 256-bit PCRs.
 #include <Library/Tcg2PreUefiEventLogLib.h>
 // MU_CHANGE [END]
+// MU_CHANGE [BEGIN]- Add InsecureDeviceStateString
+#define MSFT_INSECURE_DEVICE_STATE_STRING  "Microsoft UEFI Insecure Device State"

```

Comment:
I think we can generalize this away from Microsoft, that original code that did this was old and specific to us, this is theoretically something that could be leveraged by others.